addons/website_theme_install/models/ir_module_module.py

```diff
@@ -12,22 +12,49 @@ class IrModuleModule(models.Model):
     _name = "ir.module.module"
     _description = 'Module'
     _inherit = _name
+    _theme_models = ['theme.ir.ui.view', 'theme.website.page', 'theme.website.menu', 'theme.ir.attachment']
 
     image_ids = fields.One2many('ir.attachment', 'res_id',
                                 domain=[('res_model', '=', _name), ('mimetype', '=like', 'image/%')],
                                 string='Screenshots', readonly=True)
     # for kanban view
     is_installed_on_current_website = fields.Boolean(compute='_compute_is_installed_on_current_website')
 
+    website_ids = fields.One2many('website', 'theme_id', string="Websites", help="Websites using this theme.")
+
     @api.multi
     def _compute_is_installed_on_current_website(self):
         for module in self:
             module.is_installed_on_current_website = module == self.env['website'].get_current_website().theme_id
 
     @api.multi
     def write(self, vals):
-        if self and vals.get('state') == 'installed' and self.name.startswith('theme_'):
-            _logger.info('Module %s has been loaded as theme template' % self.name)
+        if len(self) == 1 and self.name.startswith('theme_'):
+            # if the theme is now installed
+            if vals.get('state') == 'installed':
```

Comment:
In the current state this line is unnecessary diff, can put back this condition in the first condition if there's no need for another IF on "get state".